ntoskrnl/config/cmhvlist.c

```diff
@@ -160,6 +159,16 @@ CmpAddToHiveFileList(IN PCMHIVE Hive)
         return Status;
     }
 
+    Length = sizeof(OBJECT_NAME_INFORMATION) + 256 * sizeof(WCHAR);
```

Comment:
Meh, should probably ask ZwQueryObject for the length and avoid the fixed size while I'm at it.

Reply:
256 --> MAXIMUM_FILENAME_LENGTH